packages/manager/modules/iam/src/components/policyAdd/policyAdd.template.html

```diff
@@ -0,0 +1,38 @@
+<div>
+    <oui-header
+        data-heading="{{ 'iam_policy_add_title' | translate }}"
+        data-description="{{ 'iam_policy_add_description' | translate }}"
+    >
+        <!-- <Guide> -->
+        <oui-guide-menu
+            data-text="{{:: 'iam_policy_header_guide_text' | translate }}"
+        >
+            <oui-guide-menu-item
+                data-ng-repeat="guide in $ctrl.onboardingGuides track by $index"
+                data-href="{{:: guide.url }}"
+                data-external
+            >
+                <span data-ng-bind=":: guide.text"></span>
+            </oui-guide-menu-item>
+        </oui-guide-menu>
+        <!-- </Guide> -->
+    </oui-header>
+
+    <h4 data-translate="iam_policy_add_subSection_policyName"></h4>
+
+    <form novalidate name="policyForm">
+        <oui-field
+            label="{{ 'iam_policy_add_field_policyName' | translate }}"
+            size="xl"
+        >
+            <input
+                class="oui-input"
+                name="policyNameField"
+                data-type="text"
+                data-ng-model="$ctrl.policy.name"
+                data-ng-pattern="$ctrl.PATTERN_POLICY"
```

Comment:
Use the $root variable to access constants
```suggestion
                data-ng-pattern="$root.IAM.PATTERN_POLICY"
```

Reply:
It's fixed.